node/builder_miner.go

```diff
@@ -231,6 +232,7 @@ func ConfigStorageMiner(c interface{}) Option {
 		Override(new(config.SealerConfig), cfg.Storage),
 		Override(new(config.ProvingConfig), cfg.Proving),
 		Override(new(*ctladdr.AddressSelector), modules.AddressSelector(&cfg.Addresses)),
+		Override(new(*sturdydb.DB), sturdydb.NewFromConfig),
```

Comment:
NewFromConfig should be accepting the config as a parameter and it should return the constructor with the config
```suggestion
		Override(new(*sturdydb.DB), sturdydb.NewFromConfig(cfg.SturdyDB, false)),
```

The second `false` param can be for the ITest flag, which we can just set in the itest ensemble